collections/_article/dev-snapshot-godot-3-6-beta-5.md

```diff
@@ -0,0 +1,112 @@
+Tighter shadow culling reduces this workload considerably by eliminating drawcalls for shadow casters that cannot cast a shadow upon the main camera view. This involves some clever geometry, but the upshot is you should often see significantly better frame rates when using shadows.
+
+This happens automatically.
+
+### Discrete Level of Detail (LOD)
+([GH-85437](https://github.com/godotengine/godot/pull/85437))
+
+![LOD node example](/storage/blog/godot-3.6/lod_node_scene.webp)
+
+The new LOD node provides simple but powerful LOD capabilities, allowing the engine to automatically change visual representation of objects based on the distance from the camera. An example would be simplifying trees in the distance in open world games.
+
+### Mesh Merging
+([GH-61568](https://github.com/godotengine/godot/pull/61568))
+
+https://docs.godotengine.org/en/3.6/tutorials/3d/merge_groups.html
+
+![MergeGroup example](/storage/blog/godot-3.6/merge_group_house.webp)
+
+Godot 3.6 now offers a comprehensive system for mesh merging, both at design time and at runtime. OpenGL can be severely bottlenecked by drawcalls and state changes when drawing lots of objects. Now you can blast through these barriers and potentially render any number of similar objects in a single drawcall.
+
+As well as allowing you to optimize existing maps and moving objects, this also makes new procedural game types possible, as thousands of procedurally placed objects can be merged at runtime so as to render efficiently (think vegetation, rocks, furniture, houses etc).
+
+### ORM Materials
+([GH-76023](https://github.com/godotengine/godot/pull/76023))
+
+Ansraer adds support for ORM materials, which is a standard format where occlusion, roughness and metallic are combined into a single texture. This means these standard PBR textures can be used without modification, and rendering performance will likely be increased where they are used (compared to the old workflow).
+
+### Vertex cache optimization
+([GH-86339](https://github.com/godotengine/godot/pull/86339))
+
+In the mesh import options (e.g. obj, dae) you will find a new setting for "vertex cache optimization". This may increase rendering performance for high poly models on low end hardware.
+
+In order to take advantage of vertex cache optimization in an already completed project, simply delete the hidden ".godot" folder (which contains imported data), and this imported data (including optimized meshes) will be recreated next time you open the editor.
+
+## 2D
+Fixes to physics interpolation, and hierarchical culling, as well as performance increases.
+
+## Editor
+
+### View Selected Mesh Stats
+([GH-88207](https://github.com/godotengine/godot/pull/88207))
+
+![View mesh stats example](/storage/blog/godot-3.6/view_mesh_stats.webp)
+
+The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle counts, vertex counts and index counts for the selected meshes (and multimeshes).
```

Comment:
```suggestion
The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle, vertex, and index counts for the selected meshes (and multimeshes).
```
Or just:
```suggestion
The 3D view menu now offers a new (long overdue) option, "view selected mesh stats". This will display total triangle counts, vertex counts, and index counts for the selected meshes (and multimeshes).
```